SingularityService/src/main/java/com/hubspot/singularity/resources/DeployResource.java

```diff
@@ -134,4 +149,33 @@ public SingularityRequestParent cancelDeploy(
 
     return fillEntireRequest(requestWithState);
   }
+
+  @POST
+  @Path("/deploy/{deployId}/request/{requestId}")
```

Comment:
or just do `/deploys/update` since both `requestId` and `deployId` are fields in `SingularityUpdatePendingDeployRequest`